File: tests/fixtures/anti-slop/acceptance/csharp/Negative.cs
using System;

public sealed class NegativeItem
{
    public void Read() { }
}

public static class Negative
{
    public static NegativeItem DirectValue(NegativeItem value)
    {
        return value;
    }

    public static System.Reflection.MethodInfo VariableReflection(string name)
    {
        return typeof(NegativeItem).GetMethod(name)!;
    }
}

File: tests/fixtures/anti-slop/acceptance/java/Negative.java
final class NegativeItem {
    int read() {
        return 1;
    }
}

public class Negative {
    static NegativeItem directValue(NegativeItem value) {
        return value;
    }

    static java.lang.reflect.Method variableReflection(String name) throws Exception {
        return NegativeItem.class.getDeclaredMethod(name);
    }
}
